octopoes/octopoes/repositories/ooi_repository.py

```diff
@@ -292,7 +304,10 @@ def load_bulk(self, references: set[Reference], valid_time: datetime) -> dict[st
         return {ooi.primary_key: ooi for ooi in oois}
 
     def load_bulk_as_list(self, references: set[Reference], valid_time: datetime) -> list[OOI]:
-        query = generate_pull_query(FieldSet.ALL_FIELDS, {self.pk_prefix: list(map(str, references))})
+        if not references:
+            return []
+
+        query = Query().where_in(OOI, id=references).pull(OOI, fields="[* {:_reference [*]}]")
```

Comment:
I'd rather see the extra referenced objects loaded optionally. If you look at the queries and datasets that we encode/decode and transport over the various api's it becomes clear we need to be much more precise in our resultsets as in many cases we don't use most of what we return.